modules/silicon_design/scripts/build/images/provision.sh

```diff
@@ -60,3 +61,12 @@ cp ${PROVISION_DIR}/papermill-launcher /usr/local/bin/
 chmod +x /usr/local/bin/papermill-launcher
 
 echo "DaisySuccess: done"
+
+echo "Cloning CFU-Playground repository"
```

Comment:
this need to happen before `DaisySuccess: done`

Reply:
ah okay i can fix this